app/display.py

```diff
@@ -0,0 +1,23 @@
+from app.book import Book
+
+
+class BookDisplay:
+    @staticmethod
+    def display(book: Book, display_type: str) -> None:
+        if display_type == "console":
+            print(book.content)
+        elif display_type == "reverse":
+            print(book.content[::-1])
+        else:
+            raise ValueError(f"Unknown display type: {display_type}")
+
+    @staticmethod
+    def print_book(book: Book, print_type: str) -> None:
+        if print_type == "console":
```

Comment:
Similar to the issue above, the method print_book is not properly adhering to Open-Closed principle. We should refactor this to use a strategy pattern or polymorphism to handle different print types.